src/dotnet-setversion/dotnet-setversion.csproj

```diff
@@ -14,7 +14,7 @@
     <RepositoryUrl>git@github.com:TAGC/dotnet-setversion.git</RepositoryUrl>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="CommandLineParser" Version="2.2.1" />
-    <PackageReference Include="System.Text.Json" Version="4.6.0" />
```

Comment:
Just for the sake of having it documented, how did you determine the version range for `System.Text.Json` here?

Reply:
netcoreapp2.1 (used in dotnet-setversion.csproj) doesn't support System.Runtime.CompilerServices.Unsafe, required for System.Text.Json 6.0.0 or later